src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/service/NotifyService.kt

```diff
@@ -49,6 +51,35 @@ class NotifyActionPlanService(
   }
 }
 
+@Service
+class NotifyAppointmentService(
+  @Value("\${notify.templates.appointment-not-attended}") private val appointmentNotAttendedTemplateID: String,
+  @Value("\${interventions-ui.baseurl}") private val interventionsUIBaseURL: String,
+  @Value("\${interventions-ui.locations.appointment-not-attended}") private val interventionsUIAppointmentNotAttendedLocation: String,
+  private val emailSender: EmailSender,
+  private val hmppsAuthService: HMPPSAuthService,
+) : ApplicationListener<AppointmentEvent>, NotifyService {
+
+  @AsyncEventExceptionHandling
+  override fun onApplicationEvent(event: AppointmentEvent) {
+    when (event.type) {
+      AppointmentEventType.NO_ATTENDANCE -> {
+        val userDetails = hmppsAuthService.getUserDetail(event.appointment.actionPlan.referral.createdBy)
```

Comment:
i'm sure we talked about this but i can't remember the outcome - do we want the notification to go to the PP who created the referral, or the one who sent it? 

either way, we should ensure we are consistent with these types of notifications. could we add a helper method to the referral for 'get responsible pp' or something?

Reply:
nitpick: i would also rename this variable so it's clear who's user details these are.